main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-steputils/v2/stepenv"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-utils/v2/pathutil"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/step"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/xcodebuild"
+)
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	logger := log.NewLogger()
+	step := createStep(logger)
```

Comment:
Not sure how important this is, but GoLand is warning me that

> Variable 'step' collides with imported package name

Maybe worth renaming? 🤷‍♂️

<img width="417" alt="Screen Shot 2022-06-06 at 7 16 44 AM" src="https://user-images.githubusercontent.com/923876/172159330-03c8a1f0-ee30-48b5-970e-a5c31613bc50.png">

Reply:
I was thinking about this, but could not come up with a good alternative name.
`Step` is renamed to `XcodebuildTest` in: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/commits/c01944930ba2ff613e8b9538c938101e7984fba9 .
Let me know what you think.

Reply:
Yep, I think that works just fine. I've taken to adding `Step` to the end of the name (eg. `AndroidInstrumentedTestStep`), but I wouldn't say that's necessary.